### DISCUSSION TITLE
Issues installing ruby due to no clear equivalent for zlib1g-dev/zlib-devel

### DISCUSSION THREAD

**Discussion by TheRoyalTnetennba** [Homebrew/discussions#5834] [Linux] [answered]
### Additional Info

- [ ] I am using Windows Subsystem for Linux

### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.4.13
ORIGIN: https://github.com/Homebrew/brew
HEAD: 6ea9df68df4868ce12ea7c51c85b57dce514f2e8
Last commit: 7 days ago
Branch: stable
Core tap JSON: 30 Dec 04:51 UTC
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0
HOMEBREW_EDITOR: /usr/bin/nano
HOMEBREW_MAKE_JOBS: 16
HOMEBREW_NO_AUTO_UPDATE: set
SUDO_ASKPASS: /usr/libexec/openssh/gnome-ssh-askpass
Homebrew Ruby: 3.3.6 => /var/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.6/bin/ruby
CPU: 16-core 64-bit zen4
Clang: 19.1.6
Git: 2.47.1 => /bin/git
Curl: 8.9.1 => /bin/curl
Kernel: Linux 6.11.10-304.bazzite.fc41.x86_64 x86_64 GNU/Linux
OS: Bazzite 41 (FROM Fedora Silverblue) (Holographic)
Host glibc: 2.40
/usr/bin/gcc: 14.2.1
/usr/bin/ruby: N/A
glibc: N/A
gcc@11: N/A
gcc: 14.2.0_1
xorg: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Not exactly sure where to post this (here vs ruby-build vs ?), but basically struggling to get ruby working due to (I think anyway) the lack of a zlib1g-dev/zlib-devel equivalent in homebrew.

- I can install zlib and rbenv w/ brew just fine. 
- I can even install ruby (mri 3.2.2), but running `gem install bundler` fails w/ `kernel_require.rb>:85:in 'require': cannot load such file -- zlib (LoadError)`
- Pretty much all results for this refer to either Xcode, apt zlib1g-dev, or dnf zlib-devel.
- I imagine this could probably be "solved" w/ `rpm-ostree install zlib-devel` but I'm hoping there's a  better way to resolve this. 

Feels like the people here might have the right intersection of expertise to help?

**Comment by carlocab** [comment, marked as answer]
The `zlib` formula should also provide the files that `zlib-devel` does. Not sure why you're having trouble with `gem install bundler`, though.

**Comment by TheRoyalTnetennba** [comment]
Thanks for helping me narrow it down. Posted on the [ruby-build](https://github.com/rbenv/ruby-build/issues/2487) and got a working solution there.
